runtime/web/shell.html

```diff
@@ -158,7 +158,6 @@
     <progress id="statusProgress" value="0" max="100"></progress>
   </div>
 
-  <script src="renpy-pre.js"></script>
   <script>
     // PWA
     window.addEventListener('load', function() {
```

Comment:
don't need to do after load. Installing service worker is async function, without await it won't stuck page loading process. Just remove `addEventListener` and directlty register it.
and don't need to register every time. Register a registered service worker will trigger update check of service worker, this should only use when manually update is needed.
I would like to use this below.
```
if (navigator.serviceWorker && !navigator.serviceWorker.controller) {
    navigator.serviceWorker.register('./service-worker.js', { scope: '/' });
}
```